README.md

```diff
@@ -31,14 +38,14 @@ Also called the Empower Plant UI/UX. This project was bootstrapped with [Create
  For projects that don't
 work feel free to read their README and submit a PR that makes it work. 
 
-NOTE: `build.sh` and `run.sh` files in each project are not meant to be run direclty, use top-level `deploy.sh` instead because it sets all required environment variables.
+NOTE: `build.sh` and `run.sh` files in each project are not meant to be run directly, use top-level `deploy.sh` instead because it sets all required environment variables.
 
 If you run locally and only deploy `react` it will automatically point to `staging` backends, however if you include a backend
 projects in the command `react` will magically point to it instead of staging (still requires `&be=<backend>` url param).
 
 `deploy.sh` takes another argument `--env=<env>`, which can be either `local`, `staging` or `production`. Each value corresponds to a file in `env-config` directory. `local` is a special value, most significantly it will run all webservers locally instead of deploying to Google App Engine.
 
-`deploy.sh` does everything including validating that all required values are set in the `env-config/*.env`, that each project's
+`deploy.sh` does everything including validating that all required values are set in the `env-config/local.env`, that each project's
```

Comment:
Lets keep it the way it is because in addition to `env-config/local.env` this applies to all of 
```
env-config/staging.env
env-config/production.env
env-config/my-test-env.env
```

Reply:
That makes sense - updating now!